enioka_scan/src/main/java/com/enioka/scanner/sdk/camera/CameraBarcodeScanViewScanner.java

```diff
@@ -44,9 +44,19 @@ public void initialize(Activity ctx, ScannerInitCallback cb0, ScannerDataCallbac
         cb0.onConnectionSuccessful(this);
     }
 
+    @Override
+    public void pressScanTrigger() {
+        // continuous scanning
+    }
+
+    @Override
+    public void releaseScanTrigger() {
+        // continuous scanning
+    }
+
     @Override
     public void setDataCallBack(ScannerDataCallback cb) {
-        //
+        dataDb = cb;
```

Comment:
why is it needed?

Reply:
I assumed the empty setter was a typo, but I may be wrong